### ISSUE TITLE
Missing cm:esp32:chatter2 board definition

### ISSUE THREAD

**Issue by neutralinsomniac** [CircuitMess/Arduino-Packages#2]
I was able to compile the Chatter2-Firmware repo through Arduino IDE, but I can't upload the firmware:
```
Sketch uses 1097249 bytes (30%) of program storage space. Maximum is 3612672 bytes.
Global variables use 27460 bytes (8%) of dynamic memory, leaving 300220 bytes for local variables. Maximum is 327680 bytes.
esptool.py v3.0-dev
Serial port /dev/cu.usbserial-210
Connecting.....
Chip is ESP32-D0WDQ6 (revision 1)
Features: WiFi, BT, Dual Core, 240MHz, VRef calibration in efuse, Coding Scheme None
Crystal is 40MHz
MAC: c8:f0:9e:a0:18:68
Uploading stub...
Running stub...
Stub running...
Changing baud rate to 921600
Changed.
Configuring flash size...

A fatal error occurred: Invalid head of packet (0xE0)
Failed uploading: uploading error: exit status 2
```

I noticed that I only have "Chatter by Circuitmess" instead of "Chatter2 by Circuitmess" available as a board definition in the Arduino board dropdown.

So I tried to compile the package from the commandline, but I get this:
```
jeremy@Jeremys-MacBook-Pro Chatter2-Firmware % make
[  1%] Generating build/Chatter2-Firmware.ino.elf
Error during build: Invalid FQBN: board cm:esp32:chatter2 not found
make[2]: *** [build/Chatter2-Firmware.ino.elf] Error 1
make[1]: *** [CMakeFiles/CMBuild.dir/all] Error 2
make: *** [all] Error 2
```

Is there somewhere I can find the board definition for the Chatter2?

**Comment by neutralinsomniac** [comment]
I found an [article](https://www.magazinmehatronika.com/en/circuitmess-chatter-review-the-other-side/) (near the bottom) that talked about flash issues on MacOS with earlier versions of esptool.py. So I updated the included version (3.0-dev) with version 4.6.2, and it gets a different error:
```
Sketch uses 1097249 bytes (30%) of program storage space. Maximum is 3612672 bytes.
Global variables use 27460 bytes (8%) of dynamic memory, leaving 300220 bytes for local variables. Maximum is 327680 bytes.
esptool.py v4.6.2
Serial port /dev/cu.usbserial-110
Connecting....
Chip is ESP32-D0WDQ6 (revision v1.0)
Features: WiFi, BT, Dual Core, 240MHz, VRef calibration in efuse, Coding Scheme None
Crystal is 40MHz
MAC: c8:f0:9e:a0:18:68
Uploading stub...
Running stub...
Stub running...
Changing baud rate to 921600
Changed.

A fatal error occurred: Unable to verify flash chip connection (Packet content transfer stopped (received 7 bytes)).
Failed uploading: uploading error: exit status 2
```
any ideas?

**Comment by neutralinsomniac** [comment]
Sorry for the noise, but I fixed it. Had to change my upload baud rate from 921600 to 115200 and it flashed!
